src/catalog.h

```diff
@@ -366,10 +367,11 @@ typedef struct FormData_chunk
 {
 	int32 id;
 	int32 hypertable_id;
-	NameData schema_name;
-	NameData table_name;
 	int32 compressed_chunk_id;
+	int16 chunk_status;
 	bool dropped;
+	NameData schema_name;
+	NameData table_name;
```

Comment:
Same here regarding the ordering of old fields and where the new field is added. I do not think we should do it, especially not in a minor release.